packages/browser/src/index.ts

```diff
@@ -93,14 +98,52 @@ export default class LogtoClient {
   }
 
   protected set signInSession(logtoSignInSessionItem: Optional<LogtoSignInSessionItem>) {
-    if (!logtoSignInSessionItem) {
-      sessionStorage.removeItem(this.logtoStorageKey);
+    if (logtoSignInSessionItem) {
+      const jsonItem = JSON.stringify(logtoSignInSessionItem);
+      sessionStorage.setItem(this.logtoStorageKey, jsonItem);
+
+      return;
+    }
+
+    sessionStorage.removeItem(this.logtoStorageKey);
+  }
+
+  // eslint-disable-next-line @typescript-eslint/member-ordering
```

Comment:
should we find out what [config](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/member-ordering.md) causes the conflict and change into our preference? cc @demonzoo

Reply:
Sure, I'll take a look tomorrow

Reply:
See [discussion](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1645501572295619) on Slack.

Will remove unnecessary `eslint-disable-*` comments after updating to the new version of `@silverhand/eslint-config` ( https://github.com/silverhand-io/configs/pull/18 )

Reply:
Yep, the `member-ordering` rule has been improved in `eslint-config-xo-typescript v0.45`, with rewrites to the default sorting order, removing getters and setters from the order group, which means the eslint will not complain whereever you define those getters and setters.
Therefore, we don't have to worry about this after the upgrade.